### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by a1346054**
Fixes some issues in the test script, as well as a few other minor cleanups

### CODE REVIEW

**Comment by namjaejeon** [comment]
Thanks for your patch! And can you add prefix "exfatprogs: in subject of each patches ? and add simple patch description in patch.
@hclee Hyunchul, Could you check that test/test_fsck.sh change is proper ?

File: tests/test_fsck.sh [hunk 1]
```diff
@@ -1,75 +1,74 @@
...
-	tar -C . -xf $TESTCASE_DIR/$IMAGE_FILE.tar.xz
-	losetup -f $IMAGE_FILE
-	DEV_FILE=`losetup -j $IMAGE_FILE | awk '{print $1}' | sed 's/://g'`
+	tar -C . -xf "${TESTCASE_DIR}/${IMAGE_FILE}.tar.xz"
+	DEV_FILE=$(losetup -f "${IMAGE_FILE}" --show)
 
 	# Run fsck for repair
 	$FSCK_PROG $FSCK_OPTS $DEV_FILE
-	if [ "$?" -ne "1" ]; then
+	if [ "$?" -ne 1 ]; then
```
Comment (hclee) [inline tests/test_fsck.sh:39]: is the double quote around $? needed?

Comment (a1346054) [reply tests/test_fsck.sh:39]: It is not, $? is guaranteed to always return an integer, and shellcheck won't warn about it being unquoted.

I could have cleaned up the scripts file even more, but decided to do the least amount of changes for now.

**Comment by a1346054** [comment]
I made the changes you requested.

**Pull request merged**